tests/entity_classifier/test_entity_classifier.py

```diff
@@ -0,0 +1,153 @@
+        TestOperatorResult('PERSON'),
+    ]
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.anomyze_response',
+        return_value=anomyze_response
+    )
+
+
+@pytest.fixture
+def mocked_presidio_secret_response1(mocker):
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.analyze_response',
+        return_value=Mock()
+    )
+
+    anomyze_response = [
+        TestOperatorResult('GITHUB_TOKEN'),
+        TestOperatorResult('AWS_ACCESS_KEY'),
+    ]
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.anomyze_response',
+        return_value=anomyze_response
+    )
+
+
+@pytest.fixture
+def mocked_presidio_entity_response2(mocker):
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.analyze_response',
+        return_value=Mock()
+    )
+
+    anomyze_response = [
+        TestOperatorResult('US_SSN'),
+        TestOperatorResult('US_ITIN'),
+        TestOperatorResult('IBAN_CODE'),
+        TestOperatorResult('CREDIT_CARD'),
+    ]
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.anomyze_response',
+        return_value=anomyze_response
+    )
+
+
+@pytest.fixture
+def mocked_presidio_secret_response2(mocker):
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.analyze_response',
+        return_value=Mock()
+    )
+
+    anomyze_response = [
+        TestOperatorResult('SLACK_TOKEN'),
+        TestOperatorResult('SLACK_TOKEN'),
+        TestOperatorResult('GITHUB_TOKEN'),
+        TestOperatorResult('AWS_SECRET_KEY'),
+        TestOperatorResult('AWS_ACCESS_KEY'),
+    ]
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.anomyze_response',
+        return_value=anomyze_response
+    )
+
+
+@pytest.fixture
+def mocked_presidio_entity_response3(mocker):
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.analyze_response',
+        return_value=Mock()
+    )
+
+    anomyze_response = [
+        TestOperatorResult('US_SSN'),
+        TestOperatorResult('CREDIT_CARD'),
+    ]
+    mocker.patch(
+        'pebblo.entity_classifier.entity_classifier.EntityClassifier.anomyze_response',
+        return_value=anomyze_response
+    )
+
+
+@pytest.fixture
+def entity_classifier(mocked_objects):
+    """
+    Create an instance of the EntityClassifier class
+    """
+    return EntityClassifier()
+
+
+def test_entity_classifier_init(mocked_objects) -> None:
+    _ = EntityClassifier()
+
+
+def test_presidio_entity_classifier1(entity_classifier, mocked_presidio_entity_response1):
+    entities, total_count = entity_classifier.presidio_entity_classifier(input_text1)
+    assert entities == {'US ITIN': 1, 'US SSN': 1}
+    assert total_count == 2
+
+
+def test_presidio_secret_entity_classifier1(entity_classifier, mocked_presidio_secret_response1):
```

Comment:
What if we combine `test_presidio_entity_classifier1` and `test_presidio_secret_entity_classifier1`?

Reply:
These functions cannot be combined because they are using different mock anomyzer, but I combined all entities and secret classifiers together